contract_manager/store/chains/EVMChain/aurora_testnet.yaml

```diff
@@ -0,0 +1,6 @@
+id: aurora_testnet
+wormholeChainName: aurora
+mainnet: false
+rpcUrl: https://testnet.aurora.dev
+networkId: 1313161555
+type: EVMChain
```

Comment:
the ship may have sailed on this but i think this should be capitalized like `EvmChain`. I generally prefer to treat acronyms as a "word" for the purpose of camelcasing -- there's a whole discussion of this in the google coding conventions which i thought was persuasive. 

definitely not important though

Reply:
Yeah, you mentioned this before and I thought I fixed it everywhere. I missed this one, will make the changes 👍